client/src/components/HistoryViewer/HistoryViewer.js

```diff
@@ -246,21 +251,43 @@ class HistoryViewer extends Component {
     );
   }
 
+  renderComparisonSelectionList() {
+    const { compare: { versionFrom }, ListComponent } = this.props;
+    if (!versionFrom) {
+      return null;
+    }
+    return (
+      <ListComponent
+        versions={[versionFrom]}
+        extraClass="history-viewer__table history-viewer__table--comparison-selected"
+      />
+    );
+  }
+
   /**
    * Renders a list of versions
    *
    * @returns {HistoryViewerVersionList}
    */
   renderVersionList() {
-    const { isPreviewable, ListComponent, CompareWarningComponent } = this.props;
+    const {
+      isPreviewable,
+      ListComponent,
+      CompareWarningComponent,
+      compare,
+      compare: { versionFrom: hasVersionFrom },
+    } = this.props;
 
 
     return (
       <div className={this.getContainerClasses()}>
         <CompareWarningComponent />
+
         <div className={isPreviewable ? 'panel panel--padded panel--scrollable' : ''}>
+          {this.renderComparisonSelectionList()}
           <ListComponent
             versions={this.getVersions()}
+            withHeader={!compare || (compare && !hasVersionFrom)}
```

Comment:
Do you think `showHeader` might be a better indicator that it's a boolean prop? `withHeader` to me indicates that you can provide a header component for it to render

Reply:
OK sure, I'll rename it :)